backend/src/upload/upload.ts

```diff
@@ -65,6 +65,10 @@ export interface SurveyStructure {
   Cams: string[];
   // the available modules
   available_modules: boolean[];
+  // the survey id of the dynatest
+  dynatestId?: number;
+  // the timestamp of the survey
+  surveyDate?: string;
```

Comment:
> Also there might some places where these values can be used instead of recomputing them using `get_surveyID` or `surveyDate`

Now the  `get_surveyID` or `surveyDate` only be used once in the `find_survey`.